app/views/groups/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= render "form", group: @group %>
+<br>
+<div class="navbar is-fixed-bottom bottom">
+  <%= link_to "Back to Groups", groups_path, {method: :get, class: "button is-large is-outlined is-fullwidth bottom-btn"} %>
+</div>
```

Comment:
- [ ] Even the navbar should use only colors used in the design please address that